doc/user/data/best_practices/sizing_recommendation.yml

```diff
@@ -0,0 +1,81 @@
+
+columns:
+  - column: "Field"
+  - column: "Type"
+  - column: "Description"
+  - column: "Recommendation"
+rows:
+  - Field: "**workers**"
+    Type: int
+    Description: |
+
+      The number of timely workers in your cluster replica.
+
+    Recommendation: |
+
+      Use 1 worker per CPU core, with a minimum of 1 worker.
+
+  - Field: "**scale**"
+    Type: int
+    Description: |
+
+      The number of pods (i.e., processes) to use in a cluster replica; used to
+      scale out replicas horizontally. Each pod will be provisioned using the
+      settings defined in the size definition.
+
+    Recommendation: |
+
+      Generally, this should be set to 1. This should only be greater than 1 when a replica needs to take on limits that are greater than the maximum limits permitted on a single node.
+
+  - Field: "**cpu_exclusive**"
+    Type: bool
+    Description: |
+
+      The flag that determines if the workers should attempt to pin to a particular CPU core.
+    Recommendation: |
+
+      <a name="cpu_exclusive"></a>
+
+      Set to true **if and only if** the [`cpu_limit`](#cpu_limit) is a whole
+      number and the CPU management policy in the k8s cluster is set to static.
+
+  - Field: "**cpu_limit**"
+    Type: float
+    Description: |
+
+      <a name="cpu_limit"></a>
+      The k8s limit for CPU for a replica pod in cores.
+    Recommendation: |
+
+      Prefer whole number values to enable CPU affinity. Kubernetes only allows
+      CPU Affinity for pods taking a whole number of cores (not hyperthreads).
```

Comment:
I think the hyperthread mention is incorrect here. Kubernetes doesn't really distinguish between cores and their hyperthreads.